packages/core/src/card/card.stories.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import '@cds/core/card/register.js';
+import '@cds/core/divider/register.js';
+import { getElementStorybookArgTypes } from '@cds/core/internal';
+import { html } from 'lit-html';
+import customElements from '../../dist/core/custom-elements.json';
+
+const placeholder = 'Placeholder';
+const basicContent = `
+    Message: Can you imagine what we will be downloading in another twenty years?
+    Who would have ever thought that you could record sound with digital quality fifty years ago?
+    Now we routinely download whole albums worth of music in a couple of minutes.
+  `;
+const longerContent = `
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+    Praesent volutpat tortor eget quam auctor, quis sagittis libero auctor.
+    Nulla augue ante, tincidunt sit amet semper vitae, tempus at ipsum.
+    Vestibulum elementum, turpis quis ullamcorper fermentum, elit turpis placerat ipsum, quis convallis ex nisi sit amet lacus.
+    Ut enim ipsum, tincidunt nec luctus id, pharetra id velit. Aliquam nec elit ut neque lacinia mattis id ac lorem.
+    Vivamus egestas massa nulla, ac elementum purus pretium eu. Duis ultrices nec tortor varius consectetur.
+    Maecenas convallis bibendum urna, et aliquam quam tincidunt in.
+    Nam viverra, leo vitae imperdiet ultrices, ante mi facilisis mi, ac vehicula erat nulla sed nibh.
+    Nam euismod gravida est, convallis placerat nunc pharetra sed. Nullam scelerisque dui augue, eu porta erat tempus in.
+
+    Integer egestas massa orci, id scelerisque libero lobortis vitae.
+    Vivamus aliquet sem massa, eget sodales sapien dictum eu.
+    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
+    Suspendisse non interdum sem, nec egestas nulla.
+    Duis vitae leo justo. Fusce ante enim, tincidunt nec faucibus sed, fringilla et ipsum.
+    Proin quis diam elit. Cras nec finibus libero. Sed bibendum lectus eget magna ultricies eleifend.
+    Duis ut molestie urna. Proin cursus dolor ac ex rutrum, eu condimentum elit tincidunt.
+    In a dolor faucibus, vehicula ipsum eu, viverra lorem. Aenean congue fermentum commodo.
+    Sed ex purus, consectetur mollis urna eget, vestibulum mollis dolor.
+    Donec sodales, felis sit amet gravida ornare, magna dolor ullamcorper turpis, vel facilisis odio ipsum semper sem.
+  `;
+
+export default {
+  title: 'Stories/Card',
+  component: 'cds-card',
+  argTypes: getElementStorybookArgTypes('cds-card', customElements),
+  parameters: {
```

Comment:
Something may be off with the storybook config. I see this alert at the top of the API Options pane: `This story is not configured to handle controls.` So when I try to adjust the  any of the `API Option` values its not reflected in the UI.